lib/object.h

```diff
@@ -61,4 +61,9 @@ tool_rc tpm2_util_object_load_auth(ESYS_CONTEXT *ctx, const char *objectstr,
         const char *auth, tpm2_loaded_object *outobject,
         bool is_restricted_pswd_session, tpm2_handle_flags flags);
 
```

Comment:
Please add doxygen formatted documentation for this new library function.

Reply:
Perhaps also choose a better name than load_auth2 that can imply what the function is doing.